docs/validation.rst

```diff
@@ -36,7 +36,7 @@ Model run scripts
 A run script is invoked with the following arguments:
 
 1. The output directory.
-2. The simulator name.
```

Comment:
It would be good to have examples in the docs that illustrate tags "in action"?
Similar to the sample/examples for benchmarks.

Reply:
Will add an example or two.